task/build-image-index/0.1/build-image-index.yaml

```diff
@@ -147,3 +150,59 @@ spec:
       capabilities:
         add:
           - SETFCAP
+
+  - image: quay.io/redhat-appstudio/sbom-utility-scripts-image:d4935b35eb0849841412d5d1e481a812e939f2bf
+    name: create-sbom
+    computeResources:
+      limits:
+        memory: 512Mi
+        cpu: 200m
+      requests:
+        memory: 256Mi
+        cpu: 100m
+    env:
+    - name: IMAGE_URL
+      value: $(results.IMAGE_URL.path)
+    - name: IMAGE_DIGEST
+      value: $(results.IMAGE_DIGEST.path)
+    script: |
+      INDEX_IMAGE_PULLSPEC="${IMAGE_URL}@${IMAGE_DIGEST}"
+      buildah manifest inspect "$INDEX_IMAGE_PULLSPEC" > manifest_data.json
+      python3 /scripts/index-image-sbom-script.py \
+        --image-index-url "$IMAGE_URL" \
+        --image-index-digest "$IMAGE_DIGEST" \
+        --inspect-input-file manifest_data.json
+        --output-path sbom-results.json
+    workingDir: $(workspaces.source.path)
```

Comment:
Let's not write the file into the source workspace, it shouldn't "leak" out of this task

Reply:
This Task does not use any workspaces so this won't work. Let's please also not add workspaces to this Task unless it's absolutely necessary.